lib/danger-packwerk/check/default_formatter.rb

```diff
@@ -45,11 +46,18 @@ def format_offenses(offenses, repo_link, org_name)
         team_to_work_with = constant_source_package_ownership_info.owning_team ? constant_source_package_ownership_info.markdown_link_to_github_members_no_tag : 'the pack owner'
         privacy_violation_message = "- Does API in #{constant_source_package.name}/public support this use case?\n  - If not, can we work with #{team_to_work_with} to create and use a public API?\n  - If `#{constant_name}` should already be public, try `bin/packs make_public #{constant_location}`."
 
+        constant_location_url =
```

Comment:
Is there a unit test anywhere for when `format_offenses` is passed in? I only seeing it passed in as nil in existing specs

Reply:
Oh good question. Let me see.

Reply:
Nice, glad you asked. I pushed up tests. You were absolutely right that none of it was tested, neither the old nor the new url generation.